lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -30,6 +30,43 @@ def _invert_angle(angle):
     return (180 - angle) % 360
 
 
+def get_dicom_wedge_info(beam_reference, field):
+
+    for cp in field.ControlPointSequence:
+        if cp.WedgePositionSequence[0].WedgePosition == "OUT":
```

Comment:
Above this code I would recommend having the following:

```python
if len(cp.WedgePositionSequence) != 1:
    raise ValueError("Expected only one wedge sequence per control point")
```

As a rule of thumb, before indexing to `[0]` always put a test like that above it as you are inherently making an assumption about the format of the data. If that assumption is broken, you want the application to fail, kicking and screaming.

Reply:
Ahhh, yes this is a good call. I will make a change.